docs/creating_content_basic.md

```diff
@@ -95,5 +95,5 @@ You can select a SiteAccess to preview by using a dropdown in the preview screen
 ## Publishing content
 
 If you are ready to publish the Content item, in the menu, click **Publish**.
-To save the Content item as a draft to finish editing it later, click **Save**.
+To save the Content item as a draft to finish editing it later, click **Save**, or the **X** icon in the upper left corner, or the browser's **back button**.
```

Comment:
Might be just me, but that X and browser back button triggers a save seems strange (unintuitive) to me.
If that is the case, then maybe it needs further improving in 2.5, as it could then explain why customers are repeatedly confused by this.

/cc @SylvainGuittard

Reply:
@andrerom we've already discussed it with @SylvainGuittard, so PMs are aware of this problem which is most likely about to be solved in v3.0. For v2.5 it would be kinda BC break if we change this behavior, therefore we ended up with a better explanation here in the docs.

Reply:
It’s only BC break if we know customers expect it to behave like this. The  customer confusion on this seems to possible indicate otherwise? If so bug (bad behavior).

Reply:
I just think that there might be a number of customers using `X` and being conscious that it will save a draft. Anyway, it's rather a question for @arfaram as he brought this subject to the table in the first place.

Reply:
ok